website/django/data/searchtest.py

```diff
@@ -211,7 +173,7 @@ def test_request_with_release_number(self):
 
         release_number = self.existing_variant.Data_Release_id
         request = self.factory.get(
-            '/data/?format=json&order_by=Gene_Symbol&direction=ascending&page_size=20&page_num=0&search_term=%s&release=4&include=Variant_in_ENIGMA&include=Variant_in_ClinVar&include=Variant_in_1000_Genomes&include=Variant_in_ExAC&include=Variant_in_LOVD&include=Variant_in_BIC&include=Variant_in_ESP&include=Variant_in_exLOVD' % new_variant_1.Genomic_Coordinate_hg38)
+            '/data/?format=json&order_by=Gene_Symbol&direction=ascending&page_size=20&page_num=0&search_term=&release=%s&include=Variant_in_ENIGMA&include=Variant_in_ClinVar&include=Variant_in_1000_Genomes&include=Variant_in_ExAC&include=Variant_in_LOVD&include=Variant_in_BIC&include=Variant_in_ESP&include=Variant_in_exLOVD' % new_variant_1.Genomic_Coordinate_hg38)
```

Comment:
this is off -- it's passing the genomic coordinate in as the release number, which shouldn't work. update this test so that it asks for the release_number. you'll need to update the assertion as well so that it returns the variant with the release number expected